### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by ricardomiguel-os** [flux-iac/tofu-controller#1298] [General]
Hi,

Sorry if this is not the right place to ask this question.

Do you know when the next RC version will be released?

I'm asking because the older versions, like 0.14.1, had some critical security vulnerabilities that were fixed on v0.16.0.
But the current RC 0.16.0-rc.4 has a bug in the metrics that was already solved.

I understand that you are maintaining this project for free. Is there a release plan or something similar?

Thanks for all the good work on this project

**Comment by artem-nefedov** [comment]
I would like to know this as well.
The repo seems relatively active, but there hasn't been a release in a while.

**Comment by fnicolelli-ls** [comment]
Hello team 👋  
we're about to start a PoC as we find this project very interesting for our use case and we would like to know what are the plans for the future (i saw that you removed the roadmap from the README.md file).

Thanks a lot and a big thank you for the awesome job that you're doing!

**Comment by creatorKoo** [comment]
When will it be released? I want to migrate to tofu, but there hasn’t been a release for over a year. I think this is an issue.
Is the project dead?

**Comment by akselleirv** [comment]
Hello everyone 👋
I've only been working on this project during my day job and avoiding it after work to prevent burnout. So far, I haven't been able to dedicate time to this project during working hours, but I might be able to do so now.

Another challenge is that the other maintainers are quite busy, which means PR reviews can sometimes take a while. It would be great to find a more sustainable solution for this project.

**Reply by akselleirv** [reply]
Made new release now: https://github.com/flux-iac/tofu-controller/releases/tag/v0.16.0-rc.5

Will create a v0.17.0 release after people have tested the RC.

**Comment by artem-nefedov** [comment]
@akselleirv How about making a release based on the current main? There were quite a few changes since the last one.

**Reply by akselleirv** [reply]
Hi @artem-nefedov, I’m working on a new release and hopefully the release pipeline will work after this PR https://github.com/flux-iac/tofu-controller/pull/1507 is merged.
